stubs/python/build.gradle

```diff
@@ -58,17 +58,31 @@ task fillRunProtocScript {
     }
 }
 
+def apiVersion = project.project(":api").property("version").toString()
```

Comment:
I'm not sure but I think if we put the version in the top level gradle.properties, it will be used in subprojects. Can you check if that's true?